docs/UserGuide.md

```diff
@@ -188,7 +190,7 @@ Finds passengers whose names contain any of the given keywords.
 
 Deletes the specific passenger from the GME terminal.
 
-**Format:** `delete INDEX [INDEX INDEX...]`
+**Format:** `delete INDEX [OPTIONAL_INDEX OPTIONAL_INDEX...]`
```

Comment:
I'm not sure if optional_index is necessary because the [] notation already indicates optional variables?